### PR TITLE
Pv refutation penalty

### PR DESCRIPTION
**Pull request by VoyagerOne**
Extra penalty for PV move in previous ply when it gets refuted.

STC:
LLR: 4.49 (-2.94,2.94) [-1.50,4.50]
Total: 41094 W: 7889 L: 7620 D: 25585

LTC:
LLR: 2.95 (-2.94,2.94) [0.00,6.00]
Total: 12304 W: 1967 L: 1811 D: 8526

### CODE REVIEW

**Comment by mcostalba** [comment]
Congratulations!

Do you mind taking care of at least the basic formatting (empty lines,
wrong indent, etc..). Thanks.

**Comment by VoyagerOne** [comment]
Sure...if you don't mind can you approve tests that I have currently sitting in the queue.

**Comment by mcostalba** [comment]
Done.

**Comment by VoyagerOne** [comment]
Thanks...good deal.

File: src/search.cpp [hunk 1]
```diff
@@ -826,6 +826,7 @@ namespace {
       }
       else
           ++moveCount;
+          ss->moveCount = moveCount;
```
Comment (glinscott) [inline src/search.cpp]: The indentation and the brackets aren't matching up here.  Is this intended to only execute in the else branch?

**Comment by glinscott** [comment]
Congratulations!!  One small comment, but otherwise the idea looks really nice.

**Comment by VoyagerOne** [comment]
@glinscott 
Thanks.
Fixed the indentation.

**Comment by mstembera** [comment]
Congrats!

**Comment by zamar** [comment]
Nice idea. approved.

@glinscott: Please commit this as well when you get a chance... I'm not back home until Monday.

**Pull request closed without merging**